### ISSUE TITLE
Respect the up axis setting

### ISSUE THREAD

**Issue by Herst** [rmx/threejs-collada#20]
Sadly many exporters like the OpenCollada one fpr 3ds Max  (the built-in doesn't work for other reasons for me) don't offer any setting for this.

So just like the original Collada loader a setting should be offered. Currently, the up_axis setting is currently being ignored, I don't know what the official three.js stance is but I would guess that the norm should be that the y axis is always up just like everywhere in real-time graphics so should this feature be implemented than the flipping should be done automatically depending on this setting.

**Comment by Herst** [comment]
I personally took now a totally different route and I am converting to the JSON format now. Conversion using https://github.com/mrdoob/three.js/tree/master/utils/converters/fbx works well, just like exporting from Blender using the three.js plugin (there a flipping flipping setting is offered).

**Comment by crobi** [comment]
Generally, I'm not a fan of doing any configurable preprocessing - if the modeling application can do some operation (in this case, rotate the scene), then there doesn't need to be a setting for it in the loader. In this case however, I think it would make sense to respect the `up_axis` as defined in the COLLADA file.

That being said, this loader is currently not maintained, so don't expect this being implemented any time soon.

**Comment by Herst** [comment]
Yeah, I read about this and I also agree what you wrote in the three.js bug tracker; Collada should be especially avoided for three.js unless there are no alternatives.

(Sadly, from what I've seen it seems to be only format from which 3dsMax can export while retaining the names and then being usable in three.js [I need that for meta data since one can forget about userData even more].)

Still thanks for this loader.
